plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/RemoteTableNameCacheKey.java

```diff
@@ -18,12 +18,12 @@
 import static com.google.common.base.MoreObjects.toStringHelper;
 import static java.util.Objects.requireNonNull;
 
-final class RemoteTableNameCacheKey
+public final class RemoteTableNameCacheKey
```

Comment:
Please create a GitHub issue here mentioning the reason why you did it this way and what needs to be fixed before we can do it the "correct" way.

Add a TODO comment here referring to that issue so that we can eventually clean up the code instead of changing API is non-needed ways.

Reply:
Same TODO over the now public constructor.